microApply.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: nginx
+  labels:
+    app: nginx
+spec:
+  type: NodePort
+  ports:
+    - port: 80
+      name: nginx
+      targetPort: 80
+      nodePort: 32756
+  selector:
+    app: nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: improving-app
+  labels:
+    app: improving-app
+spec:
+  selector:
+    matchLabels:
+      app: improving-app
+  template:
+    metadata:
+      labels:
+        app: improving-app
+    spec:
+      containers:
+      - name: tenant
+        image: weinyopp/improving-app-tenant:latest
```

Comment:
Are there are plans to move this to a generic place down the line?

Reply:
No clue, would have to talk to Reid about finding a generic registry because local doesn't seem to work